ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -0,0 +1,1545 @@
+{
```

Comment:
<p>I commented on this section in another PR - see https://github.com/GoogleCloudPlatform/ai-platform-samples/pull/381#discussion_r658193573</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Done</p>